menu/iconmenu/IconMenu.c

```diff
@@ -214,20 +214,23 @@ void IconMenu(void) {
 		//If anybody has toggled the xpad left/right, disable the timeout.
 		if (temp!=0) {
 			temp = IoInputDword(0x8008) - COUNT_start;
+			if (temp > (0x369E99*BOOT_TIMEWAIT)) timedOut=1;
```

Comment:
I know that this stems from old code, but what is `0x369E99`? Where does it come from?
Feels like this factor should be applied in a function.

Reply:
Refactored timer related functions and constants too, hope it looks more clear now.